lib/request-proxy.js

```diff
@@ -86,158 +76,77 @@ proto.proxyReq = function proxyReq(opts) {
     var dest = opts.dest;
     var req = opts.req;
     var res = opts.res;
+
+    // Request-level overrides (see ctor).
     var maxRetries = opts.maxRetries;
     var retrySchedule = opts.retrySchedule;
     var endpoint = opts.endpoint || '/proxy/req';
-
-    var ringpop = this.ringpop;
-
     var timeout = opts.timeout ?
-        opts.timeout : ringpop.proxyReqTimeout;
+        opts.timeout : this.ringpop.proxyReqTimeout;
```

Comment:
hm. this is subject to personal preference. i usually like using `this` until i know that scope has changed. admittedly, this is error prone with longish functions. ill consider